### ISSUE TITLE
(weather) fog sync, remains from weather syncronization

### ISSUE THREAD

**Issue by Cuel** [acemod/ACE3#5634]
**ACE3 Version:** `3.12+.*`

**Mods:**
```
- CBA_A3
- ace
```

In continuation of

https://github.com/acemod/ACE3/issues/5465
https://github.com/acemod/ACE3/pull/5622

See: https://github.com/acemod/ACE3/pull/5710

**Description:**
Using `setFog` with `fogParams` syntax, e.g `0 setFog [0, 0, 0]` on the server will cause a fog sync to all clients.
Using `setFog` with an integer, e.g `0 setFog 1` will not cause a sync to clients.

stNVG uses the latter to limit view distance when NVG is active. Effectively it would break if missions or scripts are using the `fogParams` syntax (fog would jump?). If ace enforces the integer syntax it would render the functionality that was added in A3 for fog decay and fog base useless.

**Comment by Cuel** [comment]
Ran some tests, no mods at all

Two msisions that ran for ~35 minutes,, one with 0% to 100% forecast and the other one with Eden default values.

Script used 
```sqf
cl_log = {
	params [["_serverTime", 0]];
	diag_log format [
		"CUL_WEATHER {serverTime: %1, overcast: %2, overcastForecast: %3, fog: %4, fogForecast: %5, fogParams: %6, wind: %7, rain: %8, gusts: %9, rainbow: %10, waves: %11, lightnings: %12, time: %13, id: '%14'},",
		_serverTime,
		overcast,
		overcastForecast,
		fog,
		fogForecast,
		fogParams,
		wind,
		rain,
		gusts,
		rainbow,
		waves,
		lightnings,
		time,
		["SERVER", name player] select hasInterface
	];
};

if (isServer) then {
	[] spawn {
		waitUntil{time > 0};
		[time] remoteExecCall ["cl_log"];
		while {true} do {
			uiSleep 60;
			[time] remoteExecCall ["cl_log"];
		};
	};
};
```

parsed the output and put it into charts.
Results: https://cuel.github.io/

**Comment by Cuel** [comment]
Closed by #5710
